code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1,22 +1,32 @@
 
 /datum/reagent/blood
-			data = list("donor"=null,"viruses"=null,"blood_DNA"=null,"blood_type"=null,"resistances"=null,"trace_chem"=null,"mind"=null,"ckey"=null,"gender"=null,"real_name"=null,"cloneable"=null,"factions"=null)
-			name = "Blood"
-			id = "blood"
-			color = "#C80000" // rgb: 200, 0, 0
+	data = list("donor"=null,"viruses"=null,"blood_DNA"=null,"blood_type"=null,"resistances"=null,"trace_chem"=null,"mind"=null,"ckey"=null,"gender"=null,"real_name"=null,"cloneable"=null,"factions"=null)
+	name = "Blood"
+	id = "blood"
+	color = "#C80000" // rgb: 200, 0, 0
+	metabolization_rate = 5 //fast rate so it disappears fast.
```

Comment:
fast metabolization is required, because injecting blood into a mob to replenish his blood volume doesn't automatically delete the reagent, so it's probably better to get rid of the blood quickly (and nothing is lost since blood has no on_mob_life().